SECURITY.md

```diff
@@ -0,0 +1,163 @@
+  you for accidental, good faith violations of this policy;
+- Exempt from the Digital Millennium Copyright Act (DMCA), and we will not bring
+  a claim against you for circumvention of technology controls;
+- Exempt from restrictions in our Terms & Conditions that would interfere with
+  conducting security research, and we waive those restrictions on a limited
+  basis for work done under this policy; and
+- Lawful, helpful to the overall security of the Internet, and conducted in good
+  faith.
+
+You are expected, as always, to comply with all applicable laws.
+
+If at any time you have concerns or are uncertain whether your security research
+is consistent with this policy, please submit a report through one of our
+[Official Channels](#reporting-a-vulnerability) before going any further.
+
+# Scope and Supported Versions
+
+We (the community) support the 4 most recent minor versions of Knative. For more
+details on what this means, see our
+[release principles](https://knative.dev/community/contributing/mechanics/release-versioning-principles/).
+
+While we encourage you to report vulnerabilities in any Knative repo, at HEAD or
+the tip of any release branch (release-X.Y), we commit to patching
+vulnerabilities only at HEAD and in currently supported release branches.
+
+Repos with releases have the releases listed in the Github UI, e.g.:
+
+- [serving](https://github.com/knative/serving/releases)
+- [eventing](https://github.com/knative/eventing/releases)
+
+No support is provided for archived repos.
+
+# Reporting a Vulnerability
+
+Please email the
+[Knative Security list](mailto:knative-security@googlegroups.com) with
+confidential vulnerability reports. Someone from the list will get back to you
+within 3 business days. Accepted reports will be fixed within 90 calendar days
+via a minor release and/or one or more patch release(s) for minor versions that
+are both affected and supported at that point.
+
+We ask that you keep reports confidential for the full 90 days, or until the
+security list members release you from the confidentiality, whichever comes
+first, as we coordinate disclosures with Knative partners who may need some time
+after the vulnerability is addressed in Knative in order to update to their
+systems.
+
+TODO: GPG or other tools for ensuring confidentiality of the report?
+
+## What's in a report?
+
+Ideal reports are in English and include a description of the vulnerability, a
+proof-of-concept exploit demonstrating the vulnerability, and a statement of
+what this vulnerability puts at risk.
+
+The security list members may make a best-effort attempt to address reports that
+do not meet these standards.
+
+## Coordinated Vulnerability Disclosure (CVD)
+
+Knative Security engages with partners, including vendors that host
+Knative-based products, to responsibly disclose newly discovered
+vulnerabilities.
+
+For more information on CVD, please review the information provided in the
+following links:
```

Comment:
This is Istio's early disclosure policy, which has a pleasingly open criteria for inclusion etc: https://github.com/istio/community/blob/master/EARLY-DISCLOSURE.md. Maybe (as a follow-on, let's get this landed first!) we can take a similar approach.

Reply:
Added to WG notes from today.